storage/statedb/database.go

```diff
@@ -1103,3 +1104,22 @@ func (db *Database) UpdateMetricNodes() {
 		db.trieNodeCache.UpdateStats()
 	}
 }
+
+// SaveTrieNodeCacheToFile saves the current cached trie nodes to file to reuse when the node restarts
+func (db *Database) SaveTrieNodeCacheToFile(filePath string) {
+	if db.trieNodeCache == nil {
+		return
+	}
+	filePath = filePath + "/trie_node_cache_backup"
```

Comment:
I don't really get how the path is set.
So, when I save, the path will be `filePath + "/trie_node_cache_backup"`
And when I load, it should be at `dataPath + "/fastcache"`?

Reply:
@winnie-byun Below are my answer to your question. Please share your comments and let me know If I misunderstood.

- As backup operation deletes the data in the given path, I wanted to use different path for save/load, in case of data corruption. 
- For loading the data, I don't want to add another parameter for the location of fastcache, and this is the reason why the data should be located at `dataPath + "/fastcache"`
- However, it seems that `/trie_node_cache_backup` should be `/fastcache_backup`.

Reply:
I will change it to save the data to the given path and if not given, it will save the data to the `dataPath + /fastcache`

Reply:
@winnie-byun As you're on leave, I've merged this PR after reviewed by 3 reviewers. However, feel free to leave a comment or send me a message if you have any comments or opinions. I'll reflect them in another PR. 😄

Reply:
Thank you for the kind explaination. I am good with what you've done.